### PR TITLE
Detail Readme for lb soap feature

### PR DESCRIPTION
**Pull request by rashmihunt**
This PR has detail README of 'lb soap' feature with step by step example. @raymondfeng PTAL 
Once we enhance to use a datasource name instead of WSDL name, I will iterate this README to change it to mention datasource name (which will be a simple modification to this README) through another PR. For now,  I like to finish this README work item committed for this sprint.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,7 @@ Example:
 Refer to [Create a simple API](http://loopback.io/doc/en/lb3/Create-a-simple-API.html) 
 for more information.
 
+Refer to [README](./soap/README_Soap.md) for more information on 'lb soap' command.
```
Comment (bajtos) [inline README.md]: If find it confusing when a README files contains a reference to README again. My proposal:

```diff
- Refer to [README](./soap/README_Soap.md)
+ Refer to [README_Soap](./soap/README_Soap.md)
```

File: soap/README_Soap.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+The generator loads the WSDL and discovers services defined in the WSDL. It then prompts you to 
+select the service from a list of services.
+
+![lb soap:](images/loopback-soap-service.png)
+
+Once a 'service' is selected, it will discover and list bindings defined for the selected service. 
+Select a binding.
+
+![slc loopback:swagger](images/loopback-soap-binding.png)
```
Comment (raymondfeng) [inline soap/README_Soap.md]: There are a few references to `slc loopback:swagger`. I think it should be `lb soap` now.

File: soap/README_Soap.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+it really easy and productive for developers to define, build and consume APIs. Define data 
+models, connect to multiple data sources, write business logic in Node.js, glue on top of your
+existing services and data, and consume using JS, iOS & Android SDKs.
+
+## What is Web Service/SOAP/WSDL?
+
+Web services is a technology which lets applications to talk to each other independent 
+of platform and language. A web service is a software interface which describes a collection 
+of operations that can be invoked over the network through standardized XML messaging using 
+SOAP(Simple Object Access Protocol). WSDL(Web Services Description Language) is an
```
Comment (raymondfeng) [inline soap/README_Soap.md]: Add URLs to SOAP and WSDL?

**Comment by raymondfeng** [comment]
The content LGTM. 

Should it be placed into `generator-loopback/soap` instead?

**Comment by rashmihunt** [comment]
@raymondfeng I had asked that question - moving forward, whether 'lb' is going to be used or 'slc' ? and the answer was 'lb' which means, this README should be in 'loopback-cli' project. One opther point is, if this is under generator-loopback, then, the commands in this  readme should state 'slc loopback:soap' not 'lb soap' and we are moving away from 'slc loopback:soap'

File: soap/README_Soap.md (hunk 2 above)
Comment (rashmihunt) [inline soap/README_Soap.md]: @raymondfeng done.

File: soap/README_Soap.md (hunk 3 above)
Comment (rashmihunt) [inline soap/README_Soap.md]: fixed

**Comment by bajtos** [comment]
Ideally, there should be documentation in `generator-loopback` which uses `yo loopback:soap` and there should be docs in `loopback-cli` which uses `lb soap`. Considering the effort & maintenance costs, I am personally fine with keeping the docs in `loopback-cli` only:
 - we are not marketing generator-loopback and `yo loopback`
 - in the long term, we want to get rid of yeoman & generator-loopback anyways

**Comment by raymondfeng** [comment]
@rashmihunt 

> I had asked that question - moving forward, whether 'lb' is going to be used or 'slc' ? and the answer was 'lb' which means, this README should be in 'loopback-cli' project. One opther point is, if this is under generator-loopback, then, the commands in this readme should state 'slc loopback:soap' not 'lb soap' and we are moving away from 'slc loopback:'

Please note `generator-loopback` is used by both `loopback-cli` and 'strongloop`. That's the source of truth no matter which cli is used. That's why I prefer to have README in `generator-loopback` and probably have a link from `loopback-cli` to it.

Let's re-evaluate the issue later. I'm fine to have it merged for now.

**Pull request merged**